Examples.md

```diff
@@ -333,16 +333,17 @@ spec:
                   for SERVER in $CLICKHOUSE_SERVICES; do
                     if [[ "1" == "$MAKE_INCREMENT_BACKUP" ]]; then
                       LAST_FULL_BACKUP=$(clickhouse-client -q "SELECT name FROM system.backup_list WHERE location='remote' AND name LIKE '%full%' AND desc NOT LIKE 'broken%' ORDER BY created DESC LIMIT 1 FORMAT TabSeparatedRaw" --host="$SERVER" --port="$CLICKHOUSE_PORT" --user="$BACKUP_USER" $BACKUP_PASSWORD);
+                      TODAY_FULL_BACKUP=$(clickhouse-client -q "SELECT name FROM system.backup_list WHERE location='remote' AND name LIKE '%full%' AND desc NOT LIKE 'broken%' AND DATE_FORMAT(created, '%Y-%m-%d')=DATE_FORMAT(now(), '%Y-%m-%d') ORDER BY created DESC LIMIT 1 FORMAT TabSeparatedRaw" --host="$SERVER" --port="$CLICKHOUSE_PORT" --user="$BACKUP_USER" $BACKUP_PASSWORD)
                       PREV_BACKUP_NAME=$(clickhouse-client -q "SELECT name FROM system.backup_list WHERE location='remote' AND desc NOT LIKE 'broken%' ORDER BY created DESC LIMIT 1 FORMAT TabSeparatedRaw" --host="$SERVER" --port="$CLICKHOUSE_PORT" --user="$BACKUP_USER" $BACKUP_PASSWORD);
                       DIFF_FROM[$SERVER]="";
-                      if [[ "$FULL_BACKUP_WEEKDAY" == "$(date +%u)" || "" == "$PREV_BACKUP_NAME" || "" == "$LAST_FULL_BACKUP" ]]; then
+                      if [[ ("$FULL_BACKUP_WEEKDAY" == "$(date +%u)" && "" == "$TODAY_FULL_BACKUP") || "" == "$PREV_BACKUP_NAME" || "" == "$LAST_FULL_BACKUP" ]]; then
```

Comment:
Could you please clarify, which corner case you are trying to avoid here?

If I understood correctly, you tried to change `CronJob` schedule to run more than one time per day? And FULL backup created multiples time on Monday or other case?

Reply:
Yes exactly. I'm not sure how many people need to run their backups more than once a day but for example for some of our clusters, we need to run the backup corn job every hour which means with the current example on Mondays the jobs create 24 full backups.

Reply:
This requires to change approach
instead of `"$FULL_BACKUP_WEEKDAY" == "$(date +%u)" `
something like  ` "$FULL_BACKUP_HOUR" == "$(date +%H)" `

Reply:
I see yes there might be a better way to do that via the `date` command. IRL we want to be able to replicate what we had with the `watch` cmd:

```bash
clickhouse-backup watch --watch-interval=1h --full-interval=168h
```

1 incremental per hour
1 full per week

Reply:
You can't run `clickhouse-backup watch` outside pod with clickhouse data, in this case you will back up only schema because `clickhouse-backup` requires access to the same data files as `clickhouse-server`